applications/vanilla/controllers/class.discussioncontroller.php

```diff
@@ -95,6 +95,7 @@ public function index($DiscussionID = '', $DiscussionStub = '', $Page = '') {
         }
 
         $this->setData('Category', $Category);
+        $this->setData('Editor.BackLink', anchor($Category['Name'], categoryUrl($Category)));
```

Comment:
Was the removal of the the the `htmlspecialchars` here intentional?

Reply:
Nope. I forgot to add it back. Good catch.